### PR TITLE
Fix grafana pkg

### PR DESCRIPTION
**Pull request by MalloZup**
Before we merge I want to test it if everything is ok

### CODE REVIEW

File: packaging/obs/grafana-ha-cluster-dashboards/grafana-ha-cluster-dashboards.spec [hunk 1]
```diff
@@ -45,8 +44,13 @@ install -Dm644 dashboards/provider-sleha.yaml %{buildroot}%{provisioning_dir}/pr
 %defattr(-,root,root)
 %doc dashboards/README.md
 %license LICENSE
-%attr(0755,grafana,grafana) %dir %{dashboards_dir}/sleha
-%attr(0644,grafana,grafana) %config %{dashboards_dir}/sleha/*
-%attr(0644,root,root) %config %{provisioning_dir}/provider-sleha.yaml
+%attr(0755,root,root) %dir %{dashboards_dir}/sleha
+%attr(0755,root,root) %config %{dashboards_dir}/sleha/*
+%attr(0755,root,root) %config %{provisioning_dir}/provider-sleha.yaml
```
Comment (stefanotorresi) [inline packaging/obs/grafana-ha-cluster-dashboards/grafana-ha-cluster-dashboards.spec:49]: I don't think we need `755` permissions for these files, `644` is probably more appropriate.

**Review by stefanotorresi — CHANGES_REQUESTED** [review]

File: packaging/obs/grafana-ha-cluster-dashboards/grafana-ha-cluster-dashboards.spec [hunk 2]
```diff
@@ -45,8 +50,13 @@ install -Dm644 dashboards/provider-sleha.yaml %{buildroot}%{provisioning_dir}/pr
 %defattr(-,root,root)
 %doc dashboards/README.md
 %license LICENSE
-%attr(0755,grafana,grafana) %dir %{dashboards_dir}/sleha
+%attr(0755,root,root) %dir %{dashboards_dir}/sleha
 %attr(0644,grafana,grafana) %config %{dashboards_dir}/sleha/*
-%attr(0644,root,root) %config %{provisioning_dir}/provider-sleha.yaml
```
Comment (MalloZup) [inline packaging/obs/grafana-ha-cluster-dashboards/grafana-ha-cluster-dashboards.spec:50]: why do we have here root root?

File: packaging/obs/grafana-ha-cluster-dashboards/grafana-ha-cluster-dashboards.spec [hunk 3]
```diff
@@ -45,8 +50,13 @@ install -Dm644 dashboards/provider-sleha.yaml %{buildroot}%{provisioning_dir}/pr
...
+%attr(0644,grafana,grafana) %config %{provisioning_dir}/provider-sleha.yaml
+%attr(0644,grafana,grafana) %dir  %{_sysconfdir}/grafana
+%attr(0644,grafana,grafana) %dir  %{_sysconfdir}/grafana/provisioning
+%attr(0644,grafana,grafana) %dir  %{_sysconfdir}/grafana/provisioning/dashboards
+%attr(0644,grafana,grafana) %dir  %{_localstatedir}/lib/grafana
+%attr(0644,grafana,grafana) %dir  %{_localstatedir}/lib/grafana/dashboards
```
Comment (stefanotorresi) [inline packaging/obs/grafana-ha-cluster-dashboards/grafana-ha-cluster-dashboards.spec:60]: these have the correct user and group, but they are still directories:
```suggestion
%attr(0755,grafana,grafana) %dir  %{_localstatedir}/lib/grafana
%attr(0755,grafana,grafana) %dir  %{_localstatedir}/lib/grafana/dashboards
```

File: packaging/obs/grafana-ha-cluster-dashboards/grafana-ha-cluster-dashboards.spec [hunk 4]
```diff
@@ -23,15 +23,20 @@ Group:          System/Monitoring
 Url:            https://github.com/ClusterLabs/ha_cluster_exporter
 Source:         %{name}-%{version}.tar.gz
 BuildArch:      noarch
-Requires:       grafana
-BuildRequires:  grafana
+BuildRequires:  shadow
```
Comment (stefanotorresi) [inline packaging/obs/grafana-ha-cluster-dashboards/grafana-ha-cluster-dashboards.spec:26]: According to packaging guidelines, `Require(pre)` is more correct.
```suggestion
Require(pre):   shadow
```

**Review by stefanotorresi — CHANGES_REQUESTED** [review]
I think we're getting there!

**Comment by MalloZup** [comment]
@stefanotorresi ok take a look I tested rpm works

File: packaging/obs/grafana-ha-cluster-dashboards/grafana-ha-cluster-dashboards.spec [hunk 5]
```diff
@@ -47,6 +56,11 @@ install -Dm644 dashboards/provider-sleha.yaml %{buildroot}%{provisioning_dir}/pr
 %license LICENSE
 %attr(0755,grafana,grafana) %dir %{dashboards_dir}/sleha
 %attr(0644,grafana,grafana) %config %{dashboards_dir}/sleha/*
-%attr(0644,root,root) %config %{provisioning_dir}/provider-sleha.yaml
+%attr(0644,grafana,grafana) %config %{provisioning_dir}/provider-sleha.yaml
```
Comment (stefanotorresi) [inline packaging/obs/grafana-ha-cluster-dashboards/grafana-ha-cluster-dashboards.spec:59]: sorry, I probably wasn't clear with my previous comment, we don't need this change anymore! :)

```suggestion
%attr(0644,root,root) %config %{provisioning_dir}/provider-sleha.yaml
```

**Review by stefanotorresi — CHANGES_REQUESTED** [review]
all good, just one last revert!

**Review by stefanotorresi — APPROVED** [review]

**Pull request merged**